### PR TITLE
Added the Procedures directory.

### PR DESCRIPTION
**Pull request by rbeyer**
Modeled these documents after Astropy.

### CODE REVIEW

File: Procedures/Affiliated-Package-Review-Guidelines.md [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+
+### Integration with the PlanetaryPy ecosystem  (`ecointegration`)
+
+Next up, we need to check how well the package fits in to the existing PlanetaryPy
+ecosystem - does it make use of existing functionality, or does it duplicate it?
+
+<table>
+<tr>
+<td width=150><img src="https://img.shields.io/badge/Red-red.svg" alt="Red"></td>
+<td>Does not use PlanetaryPy or other affiliated packages anywhere
```
Comment (michaelaye) [inline Procedures/Affiliated-Package-Review-Guidelines.md]: should be `planetarypy` to refer to the core package name and not the project, right?

File: Procedures/Affiliated-Package-Review-Guidelines.md [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+where it should be possible, and/or uses other libraries instead.</td>
+</tr>
+<tr>
+<td><img src="https://img.shields.io/badge/Orange-orange.svg" alt="Orange"></td>
+<td>Makes an effort to use PlanetaryPy or other affiliated packages in
```
Comment (michaelaye) [inline Procedures/Affiliated-Package-Review-Guidelines.md]: `planetarypy`

File: Procedures/Affiliated-Package-Review-Guidelines.md [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+places, but still has other places where this could be done but
+isn’t</td>
+</tr>
+<tr>
+<td><img src="https://img.shields.io/badge/Green-brightgreen.svg" alt="Green"></td>
+<td>Uses PlanetaryPy or other affiliated packages wherever possible.</td>
```
Comment (michaelaye) [inline Procedures/Affiliated-Package-Review-Guidelines.md]: `planetarypy`

File: Procedures/Affiliated-Package-Review-Guidelines.md (hunk 1 above)
Comment (rbeyer) [reply Procedures/Affiliated-Package-Review-Guidelines.md]: Interesting point, I just mimicked the language here, and Astropy used "Astropy" here, not `astropy`, but I agree that the phrasing is a little off.  What about:

> Does not use PlanetaryPy core or affiliated packages ...

This way we include both 'flavors' of PlanetaryPy code.

File: Procedures/Affiliated-Package-Review-Guidelines.md (hunk 2 above)
Comment (rbeyer) [reply Procedures/Affiliated-Package-Review-Guidelines.md]: Same fix as above?

File: Procedures/Affiliated-Package-Review-Guidelines.md (hunk 3 above)
Comment (rbeyer) [reply Procedures/Affiliated-Package-Review-Guidelines.md]: Same fix as above?

File: Procedures/Affiliated-Package-Review-Guidelines.md (hunk 1 above)
Comment (michaelaye) [reply Procedures/Affiliated-Package-Review-Guidelines.md]: also fine. interesting that Astropy diverges from its own pattern there.

File: Procedures/Affiliated-Package-Review-Guidelines.md (hunk 2 above)
Comment (michaelaye) [reply Procedures/Affiliated-Package-Review-Guidelines.md]: yep

File: Procedures/Affiliated-Package-Review-Guidelines.md (hunk 3 above)
Comment (michaelaye) [reply Procedures/Affiliated-Package-Review-Guidelines.md]: yep

Comment (michaelaye) [reply Procedures/Affiliated-Package-Review-Guidelines.md]: but let me go through the rest before you start your engines for a change...

File: Procedures/Affiliated-Package-Review-Guidelines.md (hunk 1 above)
Comment (rbeyer) [reply Procedures/Affiliated-Package-Review-Guidelines.md]: Yeah.  Its a big project, I'm not surprised there are some things that aren't completely consistent.  I've made the changes in the most recent commit to this PR.

File: Procedures/Affiliated-Package-Review-Guidelines.md (hunk 3 above)
Comment (rbeyer) [reply Procedures/Affiliated-Package-Review-Guidelines.md]: Fari enough, I'll go do some other stuff and let you accumulate.

**Review by michaelaye — CHANGES_REQUESTED** [review]
only those minor changes commented. and maybe later the JSON format when we update it. but we can leave that for now, i think?

**Review by michaelaye — APPROVED** [review]

**Comment by michaelaye** [comment]
Looks good to me.

**Pull request merged**